File: src/main/java/org/example/Java/Act2.java
package org.example.Java;

import java.util.Scanner;

public class Act2 {
    Scanner scanner = new Scanner(System.in);
    public String ParOImpar() {
        System.out.println("Cual es tu numero?");
        int num = Integer.parseInt(scanner.nextLine());

        if (num % 2 == 0) {
            return "Par";
        }else {
            return "Impar";
        }
    }
}

File: src/main/java/org/example/Kotlin/Act2.kt
package org.example.Kotlin

import java.util.*

class Act2 {

    fun ParOImpar(): String {
        println("Cual es tu numero?")
        val num = readLine()?.toInt()

        return if (num!! % 2 == 0) {
            "Par"
        } else {
            "Impar"
        }
    }
}